pkg/ibgu/ibgu.go

```diff
@@ -343,3 +345,101 @@ func (builder *IbguBuilder) validate() (bool, error) {
 
 	return true, nil
 }
+
+// PullIbgu pulls existing ibgu into IbguBuilder struct.
+func PullIbgu(apiClient *clients.Settings, name, nsname string) (*IbguBuilder, error) {
```

Comment:
add this and the other functions above validate, exported functions should come before unexported ones

Reply:
Done- thank you.